app/models/like.rb

```diff
@@ -1,4 +1,4 @@
-class Like < ApplicationRecord
+class Likes < ApplicationRecord
```

Comment:
- [ ] Same case as discussed above, please rename the model to `Like`, not `Likes`.

```suggestion
class Like < ApplicationRecord
```